docs/installing-solidity.rst

```diff
@@ -321,7 +321,7 @@ The following are dependencies for all builds of Solidity:
 +===================================+=======================================================+
 | `CMake`_ (version 3.13+)          | Cross-platform build file generator.                  |
```

Comment:
If we want to be pedantic, we can also address https://github.com/ethereum/solidity/pull/13874#discussion_r1071488393, since Nuno's right about that you need a cmake that's newer than your boost for things to work properly - i.e. bump this for the windows case. But since we didn't account for that before, fine with me either way.